lib/dependency-manager-adapters/npm.js

```diff
@@ -58,6 +60,10 @@ module.exports = CoreObject.extend({
       let cleanupTasks = [rimraf(path.join(adapter.cwd, adapter.packageJSONBackupFileName)),
         rimraf(path.join(adapter.cwd, adapter.nodeModulesBackupLocation))];
 
+      if (this.useYarnLockFile) {
+        cleanupTasks.push(rimraf(path.join(adapter.cwd, adapter.yarnLockBackupFileName)));
```

Comment:
This should probably be in the main definition of `cleanupTasks` (why wouldn't we want to _always_ do this?).

Reply:
Do we really need to ensure it exists first? Doesn't `rimraf('some-path-that-doesnt-exist.txt')` resolve successfully?

Reply:
rimraf will throw an error when a file does not exist.

Reply:
Ugh, ok, sorry.